src/Events/Integrations/Plugins/Event_Tickets/Emails/Email/Ticket.php

```diff
@@ -152,20 +152,15 @@ public function include_event_links( $parent_template ): void {
 	 * @return void
 	 */
 	public function include_event_link_styles( $parent_template ): void {
-		$args = $parent_template->get_local_values();
```

Comment:
This also needs to be removed in the RSVP class, so that's also fixed there.

Reply:
Thank you!